src/main/java/duke/logic/commands/ExportLockerSelectCommand.java

```diff
@@ -0,0 +1,16 @@
+package duke.logic.commands;
+
+import duke.exceptions.DukeException;
+import duke.models.LockerList;
+import duke.models.locker.Locker;
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+public class ExportLockerSelectCommand extends Command {
+
+    @Override
+    public void execute(LockerList lockerList, Ui ui, Storage storage) throws DukeException {
+        ui.exportSelect();
```

Comment:
why are you passing ui.readCommand()?

Reply:
the read command is already being executed in duke.java

Reply:
edited in parser